File: sandro011/programaçao_basica/questao10.py
x = int(input("informe a hora: "))
y = int(input("informe os minutos: "))
z = int(input("informe os segundos: "))
hs = x * 60 * 60 
ms = y * 60
ss = z
ts = hs + ms + ss

print("total de segundos: " + str(ts))

File: sandro011/programaçao_basica/questao10.java
import java.util.Scanner; 

public class questao10 {
    public static void main (String [] args) {
        Scanner s = new Scanner(System.in);
        int x;
        int y;
        int z;

        System.out.print("informe a hora: ");
        x = s.nextInt();

        System.out.print("informe os minutos: ");
        y = s.nextInt();

        System.out.print("informe os segundos: ");
        z = s.nextInt();
        int hs = x * 60 * 60;
        int ms = y * 60;
        int ss = z;
        int ts = hs + ms + ss;
        System.out.println("total de segundos: " + ts);
    }
}


        
         

